### ISSUE TITLE
Feature Request: raw sql support in Model definition

### ISSUE THREAD

**Issue by tjwebb** [balderdashy/sails-postgresql#98]
I'm running into issues creating any non-trivial database schema using Waterline. Basic things like compound primary keys (balderdashy/waterline#109, https://github.com/balderdashy/sails-postgresql/issues/87) are not possible. I propose the ability to add raw SQL as part of the model definition that will be executed at the time the table is created. Something like [knex.raw()](http://knexjs.org/#Schema-raw) might serve as a sensible example of why this type of feature can be useful.

It's always going to be a problem that we have to wait for the upstream Waterline project to make certain features possible. I think raw SQL support allows us to not be constantly blocked on such things as we wait for features to be merged in.

**Comment by sailsbot** [comment]
Thanks for posting, @tjwebb.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by DaAwesomeP** [comment]
Any news on PLV8 support?
